src/style.css

```diff
@@ -2,0 +1,31 @@
+body {
+  margin: 0;
+}
+
+.container {
+  height: 96px;
+  width: 96px;
+  left: 22px;
```

Comment:
The `left` property is used here, but without a `position` property (like `relative`, `absolute`, or `fixed`), it won't have any effect. Consider adding a `position` property to `.container` if you intend to position it.